c2corg_api/views/user.py

```diff
@@ -372,6 +372,7 @@ def get(self):
             'name': user.name,
             'forum_username': user.forum_username,
             'id': user.id
+
```

Comment:
@ginold The version that you want to merge contains a blank line. Make sure that you have pushed the latest version.